.github/workflows/integration_test.yml

```diff
@@ -1,19 +1,21 @@
 name: Integration Tests
-on: [push]
+on: push
 jobs:
   test:
     runs-on: ubuntu-latest
     strategy:
       matrix:
         entry:
-          - { ruby: 2.7.1, concurrency: async-websocket }
+          - { ruby: 2.7, concurrency: async-websocket }
     name: test (ruby=${{ matrix.entry.ruby }}, concurrency=${{ matrix.entry.concurrency || 'none' }})
```

Comment:
It's strange to me that this is using the matrix strategy for a single run. Idk if it was just copied from the standard test workflow or if it was intentional. We could set it up like other simple workflows to pull ruby version from `.ruby-version`

Reply:
It used to have multiple concurrency models, faye-websocket and async-websocket, thus it was a matrix. Then the matrix got simpler. You can definitely collapse this if you want to.